docs/cli/index.md

```
@@ -50,6 +50,15 @@ $ bindizr record bulk-create records.json --zone <ZONE_NAME> --preview
 $ bindizr zone import <ZONE_NAME> zone.txt --preview
 ```
 
+A zone served elsewhere imports without exporting a file first —
+`--from-server` pulls the records over AXFR (the source must allow the
+transfer; its SOA and DNSSEC records are dropped since the zone keeps its
+own SOA fields and signs itself):
+
+```bash
+$ bindizr zone import <ZONE_NAME> --from-server 192.0.2.1:53 --mode replace --preview
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)